spec/index.md

```diff
@@ -2340,6 +2340,34 @@ This function returns fee percentiles, measured in millisatoshi/vbyte (1000 mill
 
 The [standard nearest-rank estimation method](https://en.wikipedia.org/wiki/Percentile#The_nearest-rank_method), inclusive, with the addition of a 0th percentile is used. Concretely, for any i from 1 to 100, the ith percentile is the fee with rank `⌈i * 100⌉`. The 0th percentile is defined as the smallest fee (excluding coinbase transactions).
 
+### IC method `fetch_logs` {#ic-fetch_logs}
+
+Given a canister ID as input provides the logs of that canister including trap messages.
+The total length of all log contents does not exceed 4KiB.
+If new logs are added that exceed the memory capacity, the oldest logs will be removed.
+Log memory persists across upgrades and it is deleted if the canister is reinstalled or uninstalled.
+The log visibility is defined in `log_visibility` field of `canister_settings`.
+
+A single log entry is a record with the following fields:
+
+- `idx` (`nat64`): the unique incremental index of an entry;
+- `timestamp_nanos` (`nat64`): the timestamp as nanoseconds since 1970-01-01 at which the entry was recorded;
+- `contents` (`blob`): the content of a log entry;
+
+:::note
+
+This method is exposed as a query and is only accessible in non-replicated mode. 
```

Comment:
> is only accessible in non-replicated mode

does it mean that composite queries can invoke `fetch_logs` as a nested call?

Reply:
From the spec perspective, I'd go for not allowing calls to `fetch_logs` as nested calls in composite query evaluation.

Reply:
Added "only users can call, not canisters, no nested composite queries".